api/managementpb/backup/artifacts.proto

```diff
@@ -48,6 +48,15 @@ message ListArtifactsResponse {
   repeated Artifact artifacts = 1;
 }
 
+message RemoveArtifactRequest {
```

Comment:
It should be `DeleteArtifactRequest` according to our naming policy.

Reply:
done